ci/requirements-demos.txt

```diff
@@ -1 +1,3 @@
 numpy==1.17.0 ; python_version >= "3.4"
+scipy==1.3.1
+nibabel==2.5.0
```

Comment:
I can tell that you haven't used `pip-compile`, because the packages aren't in sorted order. :-) Please use `pip-compile` to regenerate this file.

Reply:
regenerated